lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+  binding.pry
+  attr_reader :name, :age, :permit, :license_data
+
+    def initialize(name, age, permit = false)
+      @name = name
+      @age = age
+      @permit = permit
+      @license_data = {:written => false, :license => false, :renewed => false}
+    end
+
+    def permit?
+      @permit
+    end
+
+    def earn_permit
+      @permit = true
+    end
+    def license_data
```

Comment:
This method is not necessary since you defined this exact method using the attr_reader :license_data.